CHANGELOG.md

```diff
@@ -5,11 +5,18 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.1.1] 2020-03-04
+
+### Changed
+
+- Updated microerror to `v0.1.0.
```

Comment:
Can we make a release as a separate PR?

Reply:
Ok

Reply:
https://github.com/giantswarm/micrologger/pull/37